Makefile.dep

```diff
@@ -496,6 +496,11 @@ ifneq (,$(filter stdio_cdc_acm stdio_null stdio_uart,$(USEMODULE)))
   DISABLE_MODULE += stdio_rtt
 endif
 
+ifneq (,$(filter stdio_rtt stdio_null stdio_uart,$(USEMODULE)))
+  # stdio_cdc_acm cannot be used when another STDIO is loaded
+  DISABLE_MODULE += stdio_cdc_acm
```

Comment:
I think this doesn't work..., and it didn't really work for `stdio_rtt` either, this is how we currently disable modules:

```
# handle removal of default modules
USEMODULE += $(filter-out $(DISABLE_MODULE), $(DEFAULT_MODULE))

# process dependencies
include $(RIOTBASE)/Makefile.dep
```

which means this `DISABLE_MODULE` has no effect. Wee need to think of a fix for this.

Reply:
I know this was done on my suggestion, I'll try to think of a proper fix.

Reply:
Can you add a big comment here on why this doesn't work.

Reply:
Talking offline this should be an issue.